katalog/cert-manager/MAINTENANCE.md

```diff
@@ -34,8 +34,7 @@ References:
 01. Download upstream manifests:
 
     ```bash
-    export CERT_MANAGER_VERSION=1.14.2
-    curl --location --remote-name https://github.com/cert-manager/cert-manager/releases/download/v"${CERT_MANAGER_VERSION}"/cert-manager.yaml
+    curl --location --remote-name https://github.com/cert-manager/cert-manager/releases/latest/download/cert-manager.yaml
```

Comment:
let's keep the version explicit, it is not granted that `latest` is what you want. For example, let's say the latest minor is 1.13.0, if a bugfix is released for 1.12 GitHub will mark that release as the latest one.

Or just leave both options and let the next maintainer decide what to use.